core/src/main/java/de/danoeh/antennapod/core/util/FeedItemPermutors.java

```diff
@@ -90,6 +98,11 @@ private static int duration(@Nullable FeedItem item) {
         return (item != null && item.getMedia() != null) ? item.getMedia().getDuration() : 0;
     }
 
+    @NonNull
+    private static String itemLink(@Nullable FeedItem item) {
+        return (item != null && item.getLink() != null) ? item.getLink() : "";
```

Comment:
I think it would make sense to convert it to lower case. Otherwise, "Bxxx" will be shown before "axxx".

Reply:
Good call @ByteHamster, I've made the change and tested it. I also took the liberty to modify the syntax in the switch-case to match the rest of the cases. I was a bit apprehensive to change things that are "not mine" but getting more comfortable doing the sensible thing. Thanks for all the patient coaching.

Input files:
![Screenshot from 2022-01-03 17-00-24](https://user-images.githubusercontent.com/42972338/147989965-598a8ebb-3a2a-492f-9436-937a69b6d338.png)

Default sort when first adding local feed:
![Screenshot_1641250673](https://user-images.githubusercontent.com/42972338/147989991-86deaceb-0304-422b-a673-632f967b9173.png)

File A to Z sort:
![Screenshot_1641250688](https://user-images.githubusercontent.com/42972338/147990004-66d295e8-8b71-45a3-810b-6871486951d7.png)

Let me know if there are other changes. Cheers!